docs/network/evm-nodes.md

```diff
@@ -70,35 +91,54 @@ You can initialize the node from a snapshot, initialize it from an existing Ethe
 
 ### From a snapshot
 
-To automatically download and import a snapshot, start the node with the `--init-from-snapshot` switch and the network to use, as in this example:
+You can import a snapshot into the EVM node in any of these ways:
 
-```bash
-octez-evm-node run observer \
-  --data-dir <EVM_DATA_DIR> \
-  --network testnet \
-  --init-from-snapshot
-```
+- Manually download and import the snapshot
+- Provide the URL of the snapshot to the node
+- Allow the node to download the snapshot automatically
 
-The node can take time to download and import the snapshot.
+You must download the appropriate snapshot for the network and mode.
+Rolling and full snapshots include 1 day of complete data.
+Therefore the rolling snapshot is equivalent to the mode `rolling:1`.
+If you want a longer rolling mode, you can download an older snapshot and the EVM node computes the complete data since the snapshot was taken.
+You can also start with a node in another mode and switch to rolling mode with any number of days of complete data.
 
-To import the snapshot manually, download the snapshot from http://snapshotter-sandbox.nomadic-labs.eu/ and import it with this command:
+To download and import the snapshot manually, download the appropriate snapshot for the network and mode from http://snapshotter-sandbox.nomadic-labs.eu/ and import it with this command:
 
 ```bash
-wget http://snapshotter-sandbox.nomadic-labs.eu/etherlink-mainnet/evm-snapshot-sr1Ghq66tYK9y-latest.gz # this is for the latest mainnet etherlink snapshots, similarly there is one for testnet
-octez-evm-node snapshot import evm-snapshot-sr1Ghq66tYK9y-latest.gz \
+wget https://storage.googleapis.com/nl-sandboxes-etherlink--snapshots/etherlink-testnet/rolling/etherlink-testnet-rolling-latest.gz
+octez-evm-node snapshot import etherlink-testnet-rolling-latest.gz \
   --data-dir <EVM_DATA_DIR>
 ```
 
-You can also pass the URL of the snapshot directly to the `octez-evm-node snapshot import` command.
+You can also pass the snapshot URL to the command, as in this example:
 
-Then, run this command to start the node, passing the data directory and the network to use:
+```bash
+octez-evm-node snapshot import https://storage.googleapis.com/nl-sandboxes-etherlink--snapshots/etherlink-testnet/rolling/etherlink-testnet-rolling-latest.gz \
+  --data-dir <EVM_DATA_DIR>
+```
+
+Then, run this command to start the node, passing the data directory and the network and mode to use:
 
 ```bash
 octez-evm-node run observer \
   --network testnet \
+  --history rolling:1 \
```

Comment:
`--history` can be omitted (it’s more a safeguard to ensure the snapshot was indeed of the expected mode)

Reply:
> If you have configured the data directory and imported a snapshot, you can omit the `--history` argument from the `octez-evm-node run` command. Including this argument allows you to verify that the node is configured for the mode that you want to run the node in. The node throws an error if you try to run it in a mode that it is not configured for.